### PR TITLE
Return null instead of empty string in response in /martha_v3 [WA-243]

### PR DESCRIPTION
**Pull request by salonishah11**
Closes https://broadworkbench.atlassian.net/browse/WA-243

### CODE REVIEW

**Review by kshakir — APPROVED** [review]

File: common/helpers.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ function validateDataObjectUrl(someUrl) {
     }
 }
 
+function isNullish(value) {
+    return value === null || typeof value === "undefined";
```
Comment (evanbernstein) [inline common/helpers.js:37]: Question, are we checking if something is `null` or `undefined` or are we checking if something is `null` or the string `"undefined"`.  The latter seems surprising (but entirely plausible). If it is the string, do you know what was causing that string?

File: test/common/helpers.test.js [hunk 2]
```diff
@@ -222,3 +222,85 @@ test('getHashesMap should throw error if the checksums array contains duplicate
         'Should have throw error but didnt!'
     );
 });
+
+/**
+ * Test the convertToMarthaV3Response() function
```
Comment (evanbernstein) [inline test/common/helpers.test.js:227]: In addition to these unit tests, what is your thought about adding a test to the overall marthav3 test where something in the DRS response should come back as null in the overall response?

File: common/helpers.js (hunk 1 above)
Comment (salonishah11) [reply common/helpers.js:37]: The former. It is checking whether it is `null` or `undefined`

File: test/common/helpers.test.js (hunk 2 above)
Comment (salonishah11) [reply test/common/helpers.test.js:227]: The way mocking is set up currently, it is tricky to resolve some fields in the response to null. I will take a look at it to see if it can be done

Comment (salonishah11) [reply test/common/helpers.test.js:227]: I was able to add such a test by changing the sandbox setup for only this test

**Review by kshakir — APPROVED** [review]

File: common/helpers.js (hunk 1 above)
Comment (evanbernstein) [reply common/helpers.js:37]: Since you are testing against `undefined` and not the string I don’t think you need that function isNullish. Instead can you try: https://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-JavaScript I’m not certain though so please try and see what you see. Hope this helps

Comment (evanbernstein) [reply common/helpers.js:37]: Sorry about this. As I mentioned in Slack, I totally missed the `===` and the `typeof`. Thank you for adding the stackoverflow reference so future folks will understand why this was implemented this way!

File: test/common/helpers.test.js (hunk 2 above)
Comment (evanbernstein) [reply test/common/helpers.test.js:227]: Thanks for adding that mocked end-to-end test!

**Review by evanbernstein — APPROVED** [review]
